Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/Views/TextAttachmentView.xaml.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Threading.Tasks;
+using Xamarin.Forms;
+using Xamarin.Forms.PlatformConfiguration;
+using Xamarin.Forms.PlatformConfiguration.iOSSpecific;
+
+namespace Contoso.Forms.Puppet.Views
+{
+    public partial class TextAttachmentView : ContentPage
+    {
+        private readonly TaskCompletionSource<string> _taskCompletionSource = new TaskCompletionSource<string>();
+        private INavigation _navigation;
+
+        public TextAttachmentView()
+        {
+            InitializeComponent();
+            On<iOS>().SetUseSafeArea(true);
```

Comment:
On Android, I got

```
[MonoDroid] System.MissingMethodException: Method 'Xamarin.Forms.PlatformConfiguration.iOSSpecific.Page.SetUseSafeArea' not found.
[MonoDroid]   at Contoso.Forms.Puppet.Views.TextAttachmentView.Show (Xamarin.Forms.INavigation navegation) [0x00001] in /Users/guperrot/git/appcenter-sdk-dotnet/Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/Views/TextAttachmentView.xaml.cs:28 
[MonoDroid]   at Contoso.Forms.Puppet.CrashesContentPage+<TextAttachment>d__5.MoveNext () [0x0001a] in /Users/guperrot/git/appcenter-sdk-dotnet/Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/CrashesContentPage.xaml.cs:60 
[MonoDroid] --- End of stack trace from previous location where exception was thrown ---
[MonoDroid]   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in <896ad1d315ca4ba7b117efb8dacaedcf>:0 
[MonoDroid]   at System.Runtime.CompilerServices.AsyncMethodBuilderCore+<>c.<ThrowAsync>b__6_0 (System.Object state) [0x00000] in <896ad1d315ca4ba7b117efb8dacaedcf>:0 
[MonoDroid]   at Android.App.SyncContext+<>c__DisplayClass2_0.<Post>b__0 () [0x00000] in <e975227ac8644a30bb0866117325de0d>:0 
[MonoDroid]   at Java.Lang.Thread+RunnableImplementor.Run () [0x00008] in <e975227ac8644a30bb0866117325de0d>:0 
[MonoDroid]   at Java.Lang.IRunnableInvoker.n_Run (System.IntPtr jnienv, System.IntPtr native__this) [0x00008] in <e975227ac8644a30bb0866117325de0d>:0 
[MonoDroid]   at (wrapper dynamic-method) System.Object:da8e2d04-867b-423e-976b-69bda16cb725 (intptr,intptr)
```

Reply:
Confirm. 😕
[Platform-Specifics - Xamarin documentation](https://developer.xamarin.com/guides/xamarin-forms/platform-features/platform-specifics/):
> Note that attempting to consume a platform-specific on a platform where it is unavailable will not result in an error. Instead, the code will execute without the platform-specific being applied.

This happens even if you surround it with runtime platform check. As well as moving it to XAML.
Looks like an issue on `Xamarin.Forms` side or I'm doing something totally wrong.
As far as I understand this is a way to apply platform-specific things in common code. Because it making no sense if it should be placed on platform-specific assembly.

Actually, I want to find how to fix it instead of removing it at all. But I haven't succeeded so far.

Reply:
Worst case, you can use the `#ifdef`

Reply:
@MatkovIvan could you remove so that we can merge and after fix it on another pr? That code didn't seem important feature for us to have.